src/mpi/coll/allreduce/Makefile.mk

```diff
@@ -9,6 +9,7 @@ mpi_core_sources +=											\
 	src/mpi/coll/allreduce/allreduce_nb.c	\
 	src/mpi/coll/allreduce/allreduce_intra_recursive_doubling.c	\
 	src/mpi/coll/allreduce/allreduce_intra_reduce_scatter_allgather.c	\
-	src/mpi/coll/allreduce/allreduce_intra_smp.c	\
+	src/mpi/coll/allreduce/allreduce_intra_reduce_scatter_allgather_const_space.c       \
+        src/mpi/coll/allreduce/allreduce_intra_smp.c	\
```

Comment:
Minor nitpick here but the leading whitespace on this line is different from the one above it.